spacy/lang/ta/examples.py

```diff
@@ -23,4 +23,6 @@
     "தன்னாட்சி கார்கள் காப்பீட்டு பொறுப்பை உற்பத்தியாளரிடம் மாற்றுகின்றன",
     "நடைபாதை விநியோக ரோபோக்களை தடை செய்வதை சான் பிரான்சிஸ்கோ கருதுகிறது",
     "லண்டன் ஐக்கிய இராச்சியத்தில் ஒரு பெரிய நகரம்."
+    "என்ன வேலை செய்கிறீர்கள்?"
```

Comment:
I think this is missing commas? 🙂

```suggestion
    "லண்டன் ஐக்கிய இராச்சியத்தில் ஒரு பெரிய நகரம்.",
    "என்ன வேலை செய்கிறீர்கள்?",
```